workflow/rocoto_funcs/setup_xml.py

```diff
@@ -66,9 +66,9 @@ def setup_xml(HOMErrfs, expdir):
       lbc(xmlFile,expdir)
       prep_ic(xmlFile,expdir)
       prep_lbc(xmlFile,expdir)
-      if os.getenv("FCST_ONLY","FALSE").upper()=="FALSE":
-        ioda_bufr(xmlFile,expdir)
-        da(xmlFile,expdir)
+#      if os.getenv("FCST_ONLY","FALSE").upper()=="FALSE":
```

Comment:
We no longer support FCST_ONLY experiments?  I think we better to keep this functionality

Reply:
We still support FCST_ONLY. This is to temporally remove the DA functions that we are not ready to deal with.

Reply:
sounds reasonable